{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2021-20216"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A flaw was found in Privoxy in versions before 3.0.31. A memory leak that occurs when decompression fails unexpectedly may lead to a denial of service. The highest threat from this vulnerability is to system availability."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-400"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1923256",
          "refsource": "MISC",
          "tags": [],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1923256"
        },
        {
          "name": "https://www.openwall.com/lists/oss-security/2021/01/31/2",
          "refsource": "MISC",
          "tags": [],
          "url": "https://www.openwall.com/lists/oss-security/2021/01/31/2"
        },
        {
          "name": "https://www.privoxy.org/3.0.31/user-manual/whatsnew.html",
          "refsource": "MISC",
          "tags": [],
          "url": "https://www.privoxy.org/3.0.31/user-manual/whatsnew.html"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2021-03-25T19:18Z",
  "publishedDate": "2021-03-25T19:15Z"
}